pygama/dsp/_processors/min_max.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from numba import guvectorize
+
+
+@guvectorize(["void(float32[:], int32[:], int32[:],float32[:], float32[:])",
+              "void(float64[:], int64[:], int64[:],float64[:], float64[:] )"],
+             "(n)->(),(),(),()", nopython=True, cache=True)
+
+
+
+def min_max(wf, argmin, argmax, wf_min, wf_max, ):
+    '''
+    Finds the min, max and their time position for a waveform
+    '''
+
+    
+    wf_min[0] = wf[0]
+    wf_max[0] = wf[0]
+    argmin[0] = 0
```

Comment:
I would suggest to inititialize argmin to `argmin[0]=float(math.inf)` and argmax[0]=float(-math.inf)`

Reply:
done